src/main.rs

```diff
@@ -54,7 +58,11 @@ impl<'a> Mqtt<'a> {
     fn connect(&mut self) -> Result<(), ClientError> {
         let mut mqtt_options = MqttOptions::new(self.name, self.host, self.port);
         mqtt_options.set_keep_alive(self.keep_alive);
-        mqtt_options.set_clean_session(self.clean_session);
+        mqtt_options.set_clean_session(self.clean_session); 
```

Comment:
There's a whitespace at the end of line, please doublecheck to remove those

Reply:
I usually rely on an automatic linter.  Given this is my first time writing any rust, I had to figure out how to run it, and no lint warnings now!